vendor/github.com/containers/image/docs/atomic-signature.md

```diff
@@ -1,241 +0,0 @@
-% atomic-signature(5) Atomic signature format
```

Comment:
Please run `make vendor` instead of doing this manually.

(And IIRC that runs into OCI JSON schema validation breaking again.)